Description: Use the same name for static and shared library
 Upstream uses libeditorconfig_static.a for static library but
 d-shlibmove expects both library to have same name. This patch
 fixes it.
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Forwarded: not-needed
Last-Update: 2016-03-25

--- a/src/lib/CMakeLists.txt
+++ b/src/lib/CMakeLists.txt
@@ -49,7 +49,7 @@
 
 add_library(editorconfig_static STATIC ${editorconfig_LIBSRCS})
 set_target_properties(editorconfig_static PROPERTIES
-    OUTPUT_NAME editorconfig_static
+    OUTPUT_NAME editorconfig
     VERSION
     ${editorconfig_VERSION_MAJOR}.${editorconfig_VERSION_MINOR}.${editorconfig_VERSION_PATCH})
 
